### DISCUSSION TITLE
Kill off stalebot

### DISCUSSION THREAD

**Discussion by vs49688** [navidrome/navidrome#2231] [General]
Automatically closing older (but still valid) issues is _really_ off-putting for contributors and reporters, and frankly just annoying. I'd say more, but I really can't put it better than these:

* https://drewdevault.com/2021/10/26/stalebot.html
* https://blog.benwinding.com/github-stale-bots/

**Comment by deluan** [comment]
I understand this can be off-putting and annoying, but keep in mind that when I [announced](https://discord.com/channels/671335427726114836/671335427726114839/1074158763541532782) I'd be setting up a stale-bot, the majority of people supported the decision. I'm adding the post here for quick reference:

>  @everyone As you all know, the number of open issues in Navidrome's GitHub page is f**king crazy huge, and I constantly feel overwhelmed by it. It is not that I don't want to work on all those features and bugs, but I don't have the time to properly sort and filter them out, and certainly not the time to work on them all.

>   So I decided to implement a "stale" bot in GitHub that will automatically close issues and PRs that have not been updated in a while. It will give the option to reply to it to keep the issue open, but there are a lot of issues that were created by people that are no longer using Navidrome, and some are unreasonable feature requests or user errors that were already resolved and the GH issue was not closed.

>   The bot will be added in the coming week, and I hope this will help keep the amount of issues manageable, reducing the time spent on issues and give me more time to actually work on features that benefit everyone.

I did some research about the topic before moving forward with this, and I read the posts you linked above. I also read this one which I feel reasons more with what I'm experiencing: https://www.jeffgeerling.com/blog/2022/burden-open-source-maintainer

Anyway, here is my reasoning for taking this decision:
- I think a lot of the "issues" reported would better be posted in a discussion/support forum. When I started the project there was no Discussions option in GitHub. Now it is there and it makes more sense to move a lot of the things we have in issues to discussions.
- The stale-bot is configured to stale issues/prs with more after 180 days! Enough for an issue be considered "stale".
- It will only close any issue after 30 days of it being marked as stale.
- I can (and will) add the `keep` label to issues/prs that are really important and should not be closed. The community can also simply comment on a stale issue, and it will be kept open for more 6 months!
- The main motivation is to resurface old issues that should be tackled and close ones that are not (or never were) relevant anymore. I can't possibly spend my limited time to work on the project going through all 350 issues and making this decision on each one. That's why the bot asks for help, it requests users to confirm whether the issue is still valid or not and comment back if it should still be open.
- I know it is my fault to let things get to this point, but you know, life happens :)

I am open to feedback and suggestions on how we can improve our issue management process, but I believe that using a stale-bot is a reasonable approach. Thank you for your contributions to the Navidrome project, and I hope we can continue to work together to make it even better.
